demosplan/DemosPlanCoreBundle/Entity/Faq.php

```diff
@@ -23,7 +24,7 @@
  *
  * @ORM\Entity(repositoryClass="demosplan\DemosPlanCoreBundle\Repository\FaqRepository")
  */
-class Faq extends CoreEntity implements UuidEntityInterface
+class Faq extends CoreEntity implements UuidEntityInterface, FaqInterface
```

Comment:
I guess "implements UuidEntityInterface," is unnecessary here because FaqInterface already extends the UuidEntityInterface.
Isn't it?